Azure/sonic-buildimage
# [Netberg][Innovium] Added support for Aurora 615/715

[nba615-showplatform-211116.txt](https://github.com/Azure/sonic-buildimage/files/7576624/nba615-showplatform-211116.txt)
[nba715-showplatform-211116.txt](https://github.com/Azure/sonic-buildimage/files/7576625/nba715-showplatform-211116.txt)
Signed-off-by: Andrew Sapronov <andrew.sapronov@gmail.com>



#### Why I did it
Suport Innovium based platforms Netberg Aurora 715/615
ASIC: Innovium Teralynx IVM 55300 (715) and 55200 (615)
Pors: 32x 100G and 48x 25G + 8x 100

#### How I did it

Added specification to device/netberg directory
Added platform/innovium/sonic-platform-modules-netberg contains kernel modules, scripts and sonic_platform packages.
Modified the platform/innovium/one-image.mk and platform/innovium/rule.mk to include Aurora 715/615 related ID and files.

#### How to verify it
Build sonic 202012 branch 

Install the image on the device and verify the related components are installed and shown correctly.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
